ts/chewing-editor_en_US.ts

```diff
@@ -60,6 +65,17 @@
         <source>About Qt</source>
         <translation></translation>
     </message>
+    <!-- Dialog -->
```

Comment:
why comment here? it's going to be removed (by Qt Linguist) in the next edit anyway.

:\  was just reviewing it, and got merged...